### PR TITLE
Move to 3.0 Preview 6 SDK

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview5-011568",
+    "dotnet": "3.0.100-preview6-012264",
     "runtimes": {
       "dotnet": [
         "2.0.9",
```
Comment (dougbu) [inline global.json]: @bricelam should this be 2.1.11 now that 2.0.x is EOL?

Comment (dougbu) [reply global.json]: Decided to get this in as-is. @bricelam you can update the 2.x runtime if and when you wish.

Comment (bricelam) [reply global.json]: No, it should remain 2.0.

**Comment by bricelam** [comment]
Why doesn't maestro do this?

**Comment by dougbu** [comment]
@bricelam it does but not automatically. I did PRs over the weekend that will update Arcade appropriately but not all of them are in yet.

In any case, there's no reason to wait for Arcade or @aspnet/build if you want the latest released SDK. There are reasons to stick with _released_ SDKs, especially where source builds are done or where a consuming repo (AspNetCore for this repo) will (soon) does source builds.

**Pull request merged**